### PR TITLE
Adding new search page

### PR DESCRIPTION
**Pull request by trungddao**
- Add curator for meta data
- Make students and advisor clickable
- first attemp for [projectStudents].astro
- Make first look on techs.astro landing page

### CODE REVIEW

File: src/pages/students/[...projectStudent].astro
Comment (trungddao) [inline src/pages/students/[...projectStudent].astro:1]: IDK what to lead when clicking on student, maybe their portfolio website, LinkedIn, GitHub?

Comment (cengique) [reply src/pages/students/[...projectStudent].astro:1]: We need to create a new content collection for students and faculty, have their own metadata, and show a simple profile page for them. Below the profile, we can list the projectSearch results for the projects they were involved in as curator/student. We can discuss about this.

File: src/pages/techs.astro [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
```
Comment (cengique) [inline src/pages/techs.astro:1]: I like what you have so far. You can display the information in a table format with tech in first column and number of projects in second. https://www.agnosticui.com/docs/components/table.html

**Review by cengique — CHANGES_REQUESTED** [review]

**Review by cengique — APPROVED** [review]
approving tech table

**Pull request merged**